modules/mbedtls/Kconfig.tls-generic

```diff
@@ -513,6 +512,18 @@ config MBEDTLS_PSA_CRYPTO_LEGACY_RNG
 
 endchoice
 
+config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG_ALLOW_NON_CSPRNG
```

Comment:
Not sure about this suggestion, but this new Kconfig option could potentially be added as part of the `choice`, removing the dependency with `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG` (and putting put the dependency of `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG` on `CSPRNG_ENABLED`)

Reply:
But this would make `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG_ALLOW_NON_CSPRNG` as the 3rd option of the choice, whereas my purpose here is to make it as a "feature" of `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG`.